pylint_odoo/checkers/no_modules.py

```diff
@@ -203,10 +211,21 @@ class NoModuleChecker(BaseChecker):
             'help': 'List of methods where call to `super` is required.' +
                     'separated by a comma.'
         }),
+        ('invalid_commit', {
```

Comment:
I got it
This is a extra parameter that you can send to pylint, but in this case, you don't need send what sentences are of `commit` because is just one and is standard the name of method `commit`.
Then you can avoid use this section for this case.